runtime/foucoco/src/assets.rs

```diff
@@ -5,34 +5,3 @@ pub mod xcm_assets {
 	create_xcm_id!(RELAY, 0);
```

Comment:
These definitions are not necessarily part of this PR but I don't think that they are right.

In the asset registry XCM(1) on Foucoco refers to "rTether USD" on the Rococo Asset Hub. XCM(2) refers to the native asset of some parachain with para id (2087) – I don't know which one that it, it is not online.

Reply:
I removed them since we also don't seem to be using this xcm assets definitions.